packages/moleculer-db-adapter-mongoose/src/index.js

```diff
@@ -132,15 +106,13 @@ class MongooseDbAdapter {
 	 * @memberof MongooseDbAdapter
 	 */
 	disconnect() {
-		return new Promise(resolve => {
-			if (this.db && this.db.close) {
-				this.db.close(resolve);
-			} else if (this.conn && this.conn.close) {
-				this.conn.close(resolve);
-			} else {
-				mongoose.connection.close(resolve);
-			}
-		});
+		if (this.db && this.db.close) {
+			return this.db.close();
+		} else if (this.conn && this.conn.close) {
+			return this.conn.close();
+		} else {
+			return mongoose.connection.close();
```

Comment:
I think this row is not relevant if we don't "touch" the default connection, right?

Reply:
yes . ( on the first check, I think opening a connection with populate the default one ... but it's not the case, so this will do nothing ) . 

I'll note to remove this ( and maybe add a warning, just in case )